docs: output of COPY INTO TABLE. (#12605)
datafuselabs/databend: docs/doc/14-sql-commands/10-dml/dml-copy-into-table.md

When importing large volumes of data, such as logs, it is recommended to set both `PURGE` and `FORCE` to True. This ensures efficient data import without the need for interaction with the Meta server (updating the copied-files set). However, it is important to be aware that this may lead to duplicate data imports.
:::

### Output

> The command returns the following columns:

Reviewer: ```suggestion
COPY INTO provides a summary of the data loading results with these columns:
```

Author: COPY INTO （table）and COPY INTO （stage ）have diff schemas

| Column           | DataType | Nullable | Description                                |
|------------------|----------|----------|--------------------------------------------|

> | FILE             | VARCHAR  | No       | The relative path to the source file       |

Reviewer: ```suggestion
| FILE             | VARCHAR  | NO       | The relative path to the source file.       |
```

> | ROWS_LOADED      | INT      | NO       | Number of rows loaded from the source file |

Reviewer: ```suggestion
| ROWS_LOADED      | INT      | NO       | The number of rows loaded from the source file. |
```

> | ERRORS_SEEN      | INT      | NO       | Number of error rows in the source file    |

Reviewer: ```suggestion
| ERRORS_SEEN      | INT      | NO       | The number of error rows found in the source file.    |
```

Reviewer: By the way, can we rename ERRORS_SEEN to ERRORS_FOUND?

> | FIRST_ERROR      | VARCHAR  | YES      | First error of the source file             |

Reviewer: ```suggestion
| FIRST_ERROR      | VARCHAR  | YES      | The first error found in the source file.             |
```

> | FIRST_ERROR_LINE | INT      | YES      | Line number of the first error             |

Reviewer: ```suggestion
| FIRST_ERROR_LINE | INT      | YES      | Line number of the first error.             |
```
